userguide/payment/includes/payment-methods-flows.adoc

```diff
@@ -0,0 +1,93 @@
+=== Payment Method Flows Overview
+
+When an end-user enters his/her payment details (like card information) into a merchant website, these payment details need to be saved in Kill Bill in order for Kill Bill to be able to process recurring payments. Additionally, the payment details may need to be saved within the payment gateway so that the end user does not have to enter their payment details each time. 
+
+*Payment method flows* describe how a user's payment details gets added in Kill Bill and the payment gateway.
+
+The end-user's payment details are referred to as a *payment method* in Kill Bill. The https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L541[PaymentApi#addPaymentMethod] is the entry point for saving a payment method in Kill Bill.
+
+Kill Bill supports three main payment method flows as follows:
+
+* *Client-side tokenization* - In this case, the end user's payment details are converted into a token and sent to Kill Bill.
```

Comment:
`and sent` -> maybe `which is sent`, to be clear it's not the payment details that are sent?
Also: `payment method details` instead of `payment details`.